console/console-init/ui/src/modules/iot-device-detail/components/DeviceDetailHeader/DeviceDetailHeader.tsx

```diff
@@ -236,7 +235,7 @@ const DeviceDetailHeader: React.FunctionComponent<IDeviceDetailHeaderProps> = ({
         <DeviceDetailLayout />
         <SplitItem isFilled />
         <SplitItem className={css(styles.kebab_toggle_margin)}>
-          <Switch
+          <SwitchWithToggle
```

Comment:
I had replaced `SwitchWithToggle` with patternfly's `Switch` for an issue that is coming up again.
![device_detail](https://user-images.githubusercontent.com/23582438/88144505-a357ab80-cc16-11ea-8671-3b42b915b2cf.gif)
Even if we cancel in the dialog, the status gets changed.